Project-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,16 @@
 import React from 'react';
-import './Header.css';
+import '../../css/index.css'; // The less-compiled css for everything
+import moment from 'moment';
+
+const HeaderTitle = () => {
+  return (
+    <React.Fragment>
+      <div className="header-title">
+        <h4>Lambda School</h4>
+        <span>@LambdaSchool · {moment().format("D MMM")}</span>
```

Comment:
Good job using moment, but as discussed during standup no reason not to use the built in Date object methods from JS. :)